.github/workflows/sql-odbc-main.yml

```diff
@@ -103,7 +102,7 @@ jobs:
     - name: build-installer
       if: success()
       run: |
-        .\scripts\build_installer.ps1 Release Win32 .\src $Env:ODBC_BUILD_PATH $Env:AWS_SDK_INSTALL_PATH
+        .\scripts\build_installer.ps1 Release Win32 .\src $Env:ODBC_BUILD_PATH .\src\vcpkg_installed\x86-windows
```

Comment:
we should consider making `.\src\vcpkg_installed` an environment variable with ODBC_BUILD_PATH